packages/fxa-payments-server/src/lib/amplitude.test.ts

```diff
@@ -68,32 +68,41 @@ it('should capture error during logAmplitudeEvent', () => {
 });
 
 it('should call logAmplitudeEvent with subscription plan info', () => {
-  const plan = { plan_id: '123xyz_hourly', product_id: '123xyz' };
-  Amplitude.createSubscription_PENDING(plan);
+  const metricsData: {
+    plan_id: string;
+    product_id: string;
+    paymentProvider?: 'Stripe' | 'PayPal';
```

Comment:
had to add this due to weird typescript issue. It was complaining that `paymentProvider` below was a string and didn't match the type of `'Stripe' | 'PayPal'...` 🤷‍♂️

Reply:
We could also export the `EventProperties` type where you added this sum type, then we can use `EventProperties['paymentProvider']` here.